content/blog/how-to-create-a-ci-cd-pipeline-to-execute-automation-test-suite-with-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,139 @@
+---
+title: How to Set Up an Automation Pipeline to Achieve Observability on HPE
+  GreenLake for Containers
+date: 2022-06-09T11:05:39.047Z
+featuredBlog: false
+priority: 1
+author: Sweta Katkoria
+authorimage: /img/swetakatkoria_photo.jpg
+thumbnailimage: /img/thumbnail.jpg
+tags:
+  - hpe-greenlake
+  - Automation
+  - Grafana
+  - observability
+---
+## Introduction
+
+An HPE GreenLake Cloud Service, [HPE GreenLake for Containers](https://www.hpe.com/us/en/greenlake/containers.html), is built upon [HPE Ezmeral Runtime Enterprise](https://www.hpe.com/us/en/software/ezmeral-runtime.html) and runs on a container-based Kubernetes orchestrated infrastructure*.* Using the Clusters module of the HPE GreenLake Central dashboard, one can perform several cluster-related operations. A cluster can be created using default or custom machine blueprints, deleted, scaled up, or scaled-down using this module. A user can navigate to the HPE Ezmeral Runtime dashboard through which various applications can be deployed and containerization can be achieved. Additionally, the cluster detail page provides some interesting insights like CPU usage, memory allocation, and storage availability. Metrics like these are crucial to tracking application usage and, subsequently, determining the resources required to grow the business. The image below illustrates what I mean.
+
+![Cluster detail screen](/img/cluster-detail.jpg "Cluster detail screen")
+
+It can be useful to get historical trend data for such metrics to analyze application-specific metrics. Using past consumption data, a customer can decide if the cluster should be scaled up on some days of the year and scaled down on others. For a system administrator working on an abstract layer of application deployment, choosing the right blueprint is crucial. A graph indicating high memory usage over CPU usage might lead him to change blueprints that offer high compute resources. The purpose of this blog post is to illustrate how a person can implement the Automation Pipeline to provide observability via the use of automation tools such as Katalon (UI automation tool), CircleCI (continuous integration and continuous delivery platform), MySQL (a database provider) and Grafana (an interactive visualization web application tool that uses time-series data to create meaningful graphs).
```

Comment:
```suggestion
It can be useful to get historical trend data for such metrics to analyze application-specific metrics. Using past consumption data, a customer can decide if the cluster should be scaled up on some days of the year and scaled down on others. For a system administrator working on an abstract layer of application deployment, choosing the right blueprint is crucial. A graph indicating high memory usage over CPU usage might lead him to change blueprints that offer high compute resources. The purpose of this blog post is to illustrate how a person can implement the Automation Pipeline and provide observability via the use of automation tools such as Katalon (UI automation tool), CircleCI (continuous integration and continuous delivery platform), MySQL (a database provider) and Grafana (an interactive visualization web application tool that uses time-series data to create meaningful graphs).
```